packages/api/src/external/ehr/elation/command/process-patients-from-appointments.ts

```diff
@@ -0,0 +1,255 @@
+import ElationApi, { ElationEnv } from "@metriport/core/external/elation/index";
+import { executeAsynchronously } from "@metriport/core/util/concurrency";
+import { out } from "@metriport/core/util/log";
+import { capture } from "@metriport/core/util/notifications";
+import { errorToString, MetriportError } from "@metriport/shared";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { getCxMappingsBySource } from "../../../../command/mapping/cx";
+import { EhrSources, getLookforwardTimeRange } from "../../shared";
+import { getElationClientKeyAndSecret, getElationEnv, MAP_KEY_SEPARATOR } from "../shared";
+import { getPatientIdOrFail } from "./get-patient";
+
+dayjs.extend(duration);
+
+const delayBetweenPracticeBatches = dayjs.duration(0, "seconds");
+const lookforward = dayjs.duration(14, "days");
+const parallelPractices = 10;
+const parallelPatients = 2;
+
+type PatientAppointment = {
+  cxId: string;
+  elationPracticeId: string;
+  elationPatientId: string;
+};
+
+export async function processPatientsFromAppointments(): Promise<void> {
+  const { log } = out(`Elation processPatientIdsOrFailFromAppointments`);
+  const elationEnvironment = getElationEnv();
+
+  const { startRange, endRange } = getLookforwardTimeRange({ lookforward });
+  log(`Getting appointments from ${startRange} to ${endRange}`);
+
+  const cxMappings = await getCxMappingsBySource({ source: EhrSources.elation });
+
+  const patientAppointmentsFromGetAppointmentsByPractice: PatientAppointment[] = [];
```

Comment:
These names 😵‍💫

Couldn't we just call this `patientAppointments`?

Reply:
That's literally what it was before an you commented that you couldn't follow....

Reply:
No longer relevant.